lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
I'm okay with that change too.

On Tuesday, 2 August 2016, Joe Lencioni <notifications@github.com> wrote:

> In lib/ImportStatements.js
> <https://github.com/Galooshi/import-js/pull/321#discussion_r73089603>:
>
> > @@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
> >  import Configuration from './Configuration';
>
> I'm fine with 80 character comments and 100 character code limits. I'm not
> sure if it really matters much to adjust the linter for this, but I'd
> support that if you want to make that change.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Galooshi/import-js/pull/321/files/5138c74aaed901105f9957cf22129203e9b5de70..7b1c648f01761d447fe1b5193caffe85d5d72636#r73089603>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAjS5Tjnqd_5YPPfGo1EApvtJeYZoLNoks5qbrkagaJpZM4JU6-_>
> .
>